README.md

```diff
@@ -166,11 +196,290 @@ function doQuery (db) {
+
+Similar to paying for time in other cloud computing services you pay Bitfinex Terminal streams on an hourly basis.
+
+As a first step we need to download the Dazaar Card for the stream we are interested in. In this how-to-guide we'll fetch data from the BTCUSD orderbook stream. A final note: please follow the code tutorial and code example first before making any payments, as payment time starts counting as soon as the payment is done.
+
+```
+wget https://github.com/bitfinexcom/bitfinex-terminal/tree/master/cards/paid/orderbooks/bitfinex.terminal.btcusd.orderbook.json
+```
+
+To access the data we need to install several modules:
+
+```
+npm install dazaar @dazaar/payment-eth \
+  bitfinex-terminal-order-book bitfinex-terminal-terms-of-use
+```
+
+One particular module we install is the terms of use module. You can read the terms of use here: [https://github.com/bitfinexcom/bitfinex-terminal/tree/master/terms-of-use](https://github.com/bitfinexcom/bitfinex-terminal/tree/master/terms-of-use). Later we will pass that module to the dazaar market to accept the terms.
+
+We create a file called `test-orderbooks.js` and load the modules we just installed with npm:
+
+```js
+const dazaar = require('dazaar')
+const swarm = require('dazaar/swarm')
+const Payment = require('@dazaar/payment-eth')
+const terms = require('bitfinex-terminal-terms-of-use')
+
+const Orderbook = require('bitfinex-terminal-order-book')
+```
+
+We also load the Dazaar Card of of the data stream:
+
+```js
+const card = require('./bitfinex.terminal.btcusd.orderbook.json')
+```
+
+When we open the file `bitfinex.terminal.btcusd.orderbook.json` we can see the Dazaar Card. The `id` field refers to the data stream, and the payment section gives instructions for payment. You can see that it requires 2 cents USDT for one hour of access:
+
+```
+  "amount": 0.02,
+  "unit": "hours",
+  "interval": 1
+```
+
+
+Similar to Free Bitfinex Terminal data we set up a database and load the card. We also accept the terms of use by passing them as an argument:
+
+```js
+const market = dazaar('./dbs/terminal-orderbook-btcusd')
+const buyer = market.buy(card, { sparse: false, terms })
+```
+
+When our local client is bootstrapped, we print the payment informations to the console:
+
+```js
+buyer.ready(function () {
+  console.log('Pay to ' + Payment.tweak(buyer.key, card))
+
+  swarm(buyer)
+})
+```
+
+It will print something like `Pay to 0xaf2333...` to the console. When we pay 2 cents USDT to that address, we get one hour of access to the stream and our replication starts. But for now, we just continue to write our example program.
+
+When we made the payment, as soon as our payment is received, the feed event is triggered, and we start replicating the data. For example purposes, we also do a database query with the orderbook query api:
+
+```js
+buyer.on('feed', function () {
+  console.log('got feed')
+
+  const o = new Orderbook(buyer.feed)
+  doQuery(o)
+})
+```
```

Comment:
suggestion: mention that the feed event fires even after payment has expired with the existing data you already fetched.